src/inference/dev_api/openvino/runtime/system_conf.hpp

```diff
@@ -131,6 +131,35 @@ OPENVINO_RUNTIME_API bool with_cpu_x86_avx512_core_amx_bf16();
  */
 OPENVINO_RUNTIME_API bool with_cpu_x86_avx512_core_amx();
 
+/**
+ * @brief      Checks whether cpu_mapping Available
+ * @ingroup    ie_dev_api_system_conf
+ * @return     `True` is CPU mapping is available, `false` otherwise
+ */
+OPENVINO_RUNTIME_API bool cpu_map_available();
+
+/**
+ * @brief      Returns number of CPU cores on Linux/Windows
+ * @ingroup    ie_dev_api_system_conf
+ * @return     A table about number of CPU cores of different types defined with ColumnOfProcessorTypeTable
```

Comment:
can we describe size of this table? etc, how to understand this table

Reply:
Added two example.